src/Infrastructure/Mosaic.php

```diff
@@ -26,94 +24,47 @@
  * @link     https://github.com/swagger-api/swagger-codegen
  */
 class Mosaic 
-	extends Service
 {
 
-    /**
-     * The Base URL for this endpoint.
+	/**
+     * Array of available mosaics by name.
      *
-     * @var string
+     * @var array 
      */
-    protected $baseUrl = "/namespace/mosaic";
-
-    /**
-     * XXX
-     *
-     * @param namespace
-     * @param id         - The topmost mosaic definition database id up to which root mosaic definitions are returned.
-     *                   The parameter is optional. If not supplied the most recent mosaic definitiona are returned.
-     * @param pageSize   - The number of mosaic definition objects to be returned for each request. The parameter is
-     *                   optional. The default value is 25, the minimum value is 5 and hte maximum value is 100.
+    static public $mosaicInfos = [
+        "xpx" => [
+            "id"   => array(481110499,231112638),
```

Comment:
This shouldn't be hard coded. Please check other SDK and see how others look up the network currency.